dali/test/python/test_operator_reduce.py

```diff
@@ -187,11 +185,15 @@ def run_reduce(keep_dims, reduce_fns, batch_gen, input_type, output_type=None):
 
 
 def test_reduce():
-    reductions = [(fn.reductions.sum, np.sum), (fn.reductions.min, np.min), (fn.reductions.max, np.max)]
+    reductions = [
+        (fn.reductions.sum, np.sum), (fn.reductions.min, np.min), (fn.reductions.max, np.max)
+    ]
 
     batch_gens = [Batch1D, Batch2D, Batch3D]
     types = [
-        np.uint8, np.int8, np.uint16, np.int16, np.uint32, np.int32, np.uint64, np.int64, np.float32]
+        np.uint8, np.int8, np.uint16, np.int16, np.uint32, np.int32,
+        np.uint64, np.int64, np.float32
```

Comment:
when we're at it....
```suggestion
        np.uint8, np.int8,
        np.uint16, np.int16,
        np.uint32, np.int32,
        np.uint64, np.int64,
        np.float32
```

Reply:
done